File: src/_03_BehavioralPattern/_03_09_Stretage/java/after/FastestSpeed.java
package _03_BehavioralPattern._03_09_Stretage.java.after;

public class FastestSpeed implements Speed {
  
  @Override
  public void blueLight() {
    System.out.println("무광꽃이");
  }

  @Override
  public void redLight() {
    System.out.println("폈슴다.");
  }
  
}

File: src/_03_BehavioralPattern/_03_09_Stretage/FastestSpeed.js
import Speed from "./Speed.js";

export default class FastestSpeed extends Speed {
  constructor() {
    super();
  }

  /** @override */
  blueLight() {
    console.log("무광꽃이");
  }

  /** @override */
  redLight() {
    console.log("폈슴다.");
  }
}